generated-sample-review/SamplegenRepeatedFields.java

```diff
@@ -0,0 +1,89 @@
+//   usage: gradle run -PmainClass=com.google.cloud.examples.speech.v1p1beta1.SamplegenRepeatedFields
+
+package com.google.cloud.examples.speech.v1p1beta1;
+
+import com.google.cloud.speech.v1p1beta1.RecognitionAudio;
+import com.google.cloud.speech.v1p1beta1.RecognitionConfig;
+import com.google.cloud.speech.v1p1beta1.RecognizeRequest;
+import com.google.cloud.speech.v1p1beta1.RecognizeResponse;
+import com.google.cloud.speech.v1p1beta1.SpeechClient;
+import com.google.cloud.speech.v1p1beta1.SpeechContext;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionAlternative;
+import com.google.cloud.speech.v1p1beta1.SpeechRecognitionResult;
+import java.util.Arrays;
+import java.util.List;
+
+public class SamplegenRepeatedFields {
+  // [START samplegen_repeated_fields]
+  /*
+   * Please include the following imports to run this sample.
+   *
+   * import com.google.cloud.speech.v1p1beta1.RecognitionAudio;
+   * import com.google.cloud.speech.v1p1beta1.RecognitionConfig;
+   * import com.google.cloud.speech.v1p1beta1.RecognizeRequest;
+   * import com.google.cloud.speech.v1p1beta1.RecognizeResponse;
+   * import com.google.cloud.speech.v1p1beta1.SpeechClient;
+   * import com.google.cloud.speech.v1p1beta1.SpeechContext;
+   * import com.google.cloud.speech.v1p1beta1.SpeechRecognitionAlternative;
+   * import com.google.cloud.speech.v1p1beta1.SpeechRecognitionResult;
+   * import java.util.Arrays;
+   * import java.util.List;
+   */
+
+  /** Showing repeated fields (in request and response) */
+  public static void sampleRecognize() {
+    try (SpeechClient speechClient = SpeechClient.create()) {
+      RecognitionConfig.AudioEncoding encoding = RecognitionConfig.AudioEncoding.MP3;
+
+      // A list of strings containing words and phrases "hints"
+      String phrasesElement = "Fox in socks";
+      String phrasesElement2 = "Knox in box";
+      List<String> phrases = Arrays.asList(phrasesElement, phrasesElement2);
+      SpeechContext speechContextsElement =
+          SpeechContext.newBuilder().addAllPhrases(phrases).build();
+      List<SpeechContext> speechContexts = Arrays.asList(speechContextsElement);
+      RecognitionConfig config =
+          RecognitionConfig.newBuilder()
+              .setEncoding(encoding)
+              .addAllSpeechContexts(speechContexts)
+              .build();
+      String uri = "gs://[BUCKET]/[FILENAME]";
+      RecognitionAudio audio = RecognitionAudio.newBuilder().setUri(uri).build();
+      RecognizeRequest request =
+          RecognizeRequest.newBuilder().setConfig(config).setAudio(audio).build();
+      RecognizeResponse response = speechClient.recognize(request);
+      // Loop over all transcription results
+      for (SpeechRecognitionResult result : response.getResultsList()) {
+        // The first "alternative" of each result contains most likely transcription
+        SpeechRecognitionAlternative alternative = result.getAlternativesList().get(0);
```

Comment:
We have results and alternatives, why are we only looking at the first alternative?

Reply:
It really comes down to: what is the point of the sample?  What are we trying to teach?